### PR TITLE
Homework Week 2

### PR DESCRIPTION
**Pull request by gulyener**
This week I didn't continue with last week's homework because it was Class based and I wanted to use functional components in this one.

### CODE REVIEW

File: homework-submission/hw-todolist/src/components/TodoList/index.jsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+            }
+          >
+            <td>{todo.description}</td>
+            <td>{todo.deadline}</td>
+            {'      '}
+            <td>
+              <button onClick={() => deleteTodo(todo.id)}>Delete todo</button>
+            </td>
+            <td>
+              <button onClick={() => markTodo(todo.id, todo.done)}>Mark todo</button>
```
Comment (ArcoMul) [inline homework-submission/hw-todolist/src/components/TodoList/index.jsx]: Would be cool if the text of the 'Mark todo' button is 'Mark done' at fist and changes to 'Mark todo' once a todo is done

File: homework-submission/hw-todolist/src/components/TodoList/index.jsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+      {todos.map(todo => {
+        return (
+          <tr
+            style={
+              todo.done === true ? { textDecoration: 'line-through' } : { textDecoration: 'none' }
+            }
+          >
+            <td>{todo.description}</td>
+            <td>{todo.deadline}</td>
+            {'      '}
```
Comment (ArcoMul) [inline homework-submission/hw-todolist/src/components/TodoList/index.jsx]: Why are these spaces needed?

Comment (gulyener) [reply homework-submission/hw-todolist/src/components/TodoList/index.jsx]: You're right I deleted it:)

**Comment by gulyener** [comment]
Thank you for the review @ArcoMul. About the delete function, I couldn't see the bug you mentioned but I changed the id's just in case. I don't know if it's correct now. I also changed TodoList component to TodoTable since it's a table. Should I create Todo Items in a separate component?

**Comment by ArcoMul** [comment]
Great improvements! `uuid` did fix the bug. What happened is that some todos ended up having the same id because you used `todos.length`

**Pull request closed without merging**